src/phoenix/server/api/types/Project.py

```diff
@@ -29,7 +29,7 @@
 from phoenix.server.api.types.Trace import Trace
 from phoenix.server.api.types.ValidationResult import ValidationResult
 from phoenix.trace.dsl import SpanFilter
-from phoenix.trace.schemas import SpanID, TraceID
+from phoenix.trace.schemas import SpanID
 
 
 @strawberry.type
```

Comment:
It would help to store the `project_rowid` here as a private field. This would allow us to not have to do joins with the projects table when query the traces and spans tables.

Reply:
updated